.infra/index.ts

```diff
@@ -522,3 +524,48 @@ if (vpcNativeProvider) {
     { provider: vpcNativeProvider.provider },
   );
 }
+
+if (!isAdhocEnv) {
+  new ClickHouseSync(
+    'clickhouse-sync',
+    {
+      isAdhocEnv: isAdhocEnv,
+      namespace: namespace,
+      props: {
+        path: './config.yml',
+        keys: {
+          ...config.requireObject<{ keys: ClickHouseSyncConfig }>(
+            'clickhouseSync',
+          ).keys,
+          'database.hostname': envVars.typeormHost as string,
+          'database.port': '5432',
+          'database.password': config.require('debeziumDbPass'),
+          'database.user': config.require('debeziumDbUser'),
+          'database.server.name': name,
+          'database.dbname': name,
+        },
+        vars: {
+          ...config.requireObject<{ vars: Record<string, string> }>(
+            'clickhouseSync',
+          ).vars,
+        },
+      },
+      image: {
+        repository: 'altinity/clickhouse-sink-connector',
+        tag: '2.0.2-lt',
+      },
+      resources: {
+        // TODO: adjust resources based on the actual usage
```

Comment:
This still needed?

Reply:
Yeah, will need to deploy and see actual real usage first

Reply:
it's basically debezium under the hood with some steroids, but with only one tiny table to monitor, so we can allocate roughly 1/3 of the resources we give debezium currently.

Reply:
Yup. My plan is to come back to this once deployed for a few days/weeks to see real usage